libs/perun/utils/src/lib/perun-utils.ts

```diff
@@ -864,3 +865,120 @@ export function isMemberIndirectString(member: RichMember): string {
   }
   return 'UNKNOWN';
 }
+
+export function escapeDangerousHtml(html: string): string {
```

Comment:
I think this DOM sanitizing would be better off as a service instead of a util function. Otherwise it is nicely done.